[2019-04-29 08:58:41,664] {models.py:1359} INFO - Dependencies all met for <TaskInstance: s3_redo_sensor.s3_file_test 2019-04-28T00:00:00+00:00 [queued]>
[2019-04-29 08:58:41,666] {models.py:1359} INFO - Dependencies all met for <TaskInstance: s3_redo_sensor.s3_file_test 2019-04-28T00:00:00+00:00 [queued]>
[2019-04-29 08:58:41,666] {models.py:1571} INFO - 
--------------------------------------------------------------------------------
Starting attempt 1 of 2
--------------------------------------------------------------------------------

[2019-04-29 08:58:41,679] {models.py:1593} INFO - Executing <Task(S3KeySensor): s3_file_test> on 2019-04-28T00:00:00+00:00
[2019-04-29 08:58:41,679] {base_task_runner.py:118} INFO - Running: ['bash', '-c', 'airflow run s3_redo_sensor s3_file_test 2019-04-28T00:00:00+00:00 --job_id 109 --raw -sd DAGS_FOLDER/AWS_Challenge.py --cfg_path /tmp/tmp52nlwr1d']
[2019-04-29 08:58:42,155] {base_task_runner.py:101} INFO - Job 109: Subtask s3_file_test [2019-04-29 08:58:42,154] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-04-29 08:58:42,313] {base_task_runner.py:101} INFO - Job 109: Subtask s3_file_test [2019-04-29 08:58:42,312] {models.py:273} INFO - Filling up the DagBag from /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/dags/AWS_Challenge.py
[2019-04-29 08:58:42,351] {base_task_runner.py:101} INFO - Job 109: Subtask s3_file_test [2019-04-29 08:58:42,351] {cli.py:520} INFO - Running <TaskInstance: s3_redo_sensor.s3_file_test 2019-04-28T00:00:00+00:00 [running]> on host localhost.localdomain
[2019-04-29 08:58:42,399] {s3_key_sensor.py:88} INFO - Poking for key : s3://cscie29-data/pset5/yelp_data/yelp_subset_*.csv
[2019-04-29 08:58:42,419] {logging_mixin.py:95} INFO - [2019-04-29 08:58:42,418] {credentials.py:1046} INFO - Found credentials in shared credentials file: ~/.aws/credentials
[2019-04-29 08:58:42,817] {logging_mixin.py:95} INFO - [2019-04-29 08:58:42,816] {credentials.py:1046} INFO - Found credentials in shared credentials file: ~/.aws/credentials
[2019-04-29 08:58:43,237] {base_sensor_operator.py:113} INFO - Success criteria met. Exiting.
[2019-04-29 08:58:43,260] {base_task_runner.py:101} INFO - Job 109: Subtask s3_file_test /home/mjcarleb/anaconda3/envs/airflow/lib/python3.5/site-packages/airflow/utils/helpers.py:356: DeprecationWarning: Importing 'BashOperator' directly from 'airflow.operators' has been deprecated. Please import from 'airflow.operators.[operator_module]' instead. Support for direct imports will be dropped entirely in Airflow 2.0.
[2019-04-29 08:58:43,261] {base_task_runner.py:101} INFO - Job 109: Subtask s3_file_test   DeprecationWarning)
[2019-04-29 08:58:46,663] {logging_mixin.py:95} INFO - [2019-04-29 08:58:46,662] {jobs.py:2527} INFO - Task exited with return code 0
